tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -210,7 +210,7 @@ ROOT::Experimental::Detail::RPageSourceFile::RPageSourceFile(std::string_view nt
    const RNTupleReadOptions &options)
    : RPageSourceFile(ntupleName, options)
 {
-   fFile = std::unique_ptr<RRawFile>(RRawFile::Create(path));
+   fFile = std::unique_ptr<ROOT::Internal::RRawFile>(ROOT::Internal::RRawFile::Create(path));
```

Comment:
Shouldn't RRawFile::Create return a unique_ptr?

Reply:
Indeed, addressed in #5018

Reply:
thanks.